.env.dist

```diff
@@ -1,5 +1,11 @@
 #DEFAULT_LOCALE=en
 #SERVICE_NAME=articles
+#S3_BUCKET=bucket
+
+###> aws/aws-sdk-php-symfony ###
+AWS_KEY=not-a-real-key
```

Comment:
the names used by this library depart from the conventions (`AWS_SECRET_ACCESS_KEY`, `AWS_ACCESS_KEY_ID`)

Reply:
Something to raise upstream.

Reply:
We can change it here though.

Reply:
In fact, might work already: https://github.com/aws/aws-sdk-php/blob/f3e7263c1e3dcc4995a7b3194faa4c04f2d8f0ad/src/Credentials/CredentialProvider.php#L46-L47

Reply:
https://github.com/libero/content-store/pull/9/commits/dac87b8095d3060fb8f2ae54043765c31582d222